backend/tests/integration/mailing/test_providers_mailtrain.py

```diff
@@ -17,3 +24,21 @@ def test_mailtrain_subscribe_user_to_list(base_app, db, es_clear, vcr_cassette):
     last_name = "Lastname"
     mailtrain_subscribe_user_to_list(list_id, email, first_name, last_name)
     assert vcr_cassette.all_played
+
+
+@freeze_time(datetime(2019, 9, 17, 6, 0, 0))
+def test_set_mailtrain_campaign_in_redis(base_app, db, redis):
+    config = {"WEEKLY_JOBS_EMAIL_REDIS_KEY": "MAILTRAIN_KEY"}
+    with patch.dict(base_app.config, config):
+        html_content = "<html><a>Some HTML content</a> Blah</html>"
+        mailtrain_update_weekly_campaign_content(html_content)
+
+        expected_keys = ["date", "title", "html"]
+        expected_values = [
+            str(datetime(2019, 9, 17, 6, 0, 0).timestamp()),
```

Comment:
:+1:

Reply:
Actually when using freezetime it makes now()0 and utcnow() to return local time. So it will be flaky test depending on timezone user have :/ Like on my PC utcnow() returns 1568700000 and on travis 1568692800 :/ for freeze_time(datetime(2019, 9, 17, 6, 0, 0))

Reply:
I think you need https://docs.python.org/3/library/datetime.html#datetime.datetime.utcfromtimestamp

Reply:
Discussed IRL